src/views/Correction.vue

```diff
@@ -877,6 +895,21 @@ section p {
   color: $gray7;
 }
 
+.correction-header {
+  font-size: 18px;
+  font-weight: bold;
+  color: $gray9;
+}
+
+.detail-comment-input::placeholder {
```

Comment:
Same for line 929 below.

Reply:
Sorry about the confusion -- `:deep` or `::v-deep` are the old syntaxes. The new syntax is `deep()` but it doesn't work in all UIs yet.

Reply:
so I should not use v-deep and instead just use deep()?

Reply:
Whatever the current version of Vue/Vuetify/compiler? supports. See surrounding code.

Eventually I think it will all be `deep()` but that doesn't work yet in some UIs.